src/v2/cookbook/using-axios-to-consume-apis.md

```diff
@@ -0,0 +1,172 @@
+      errored: false
+    }
+  },
+  filters: {
+    currencydecimal (value) {
+      return value.toFixed(2)
+    }
+  },
+  mounted () {
+    axios
+      .get('https://api.coindesk.com/v1/bpi/currentprice.json')
+      .then(response => {
+        this.info = response.data.bpi
+      })
+      .catch(error => {
+        console.log(error)
+        this.errored = true
+      }).
+      .finally(() => this.loading = false)
+  }
+})
+```
+
+```html
+<div id="app">
+  <h1>Bitcoin Price Index</h1>
+
+  <section v-if="errored">
+    <p>We're sorry, we're not able to retrieve this information at the moment, please try back later</p>
+  </section>
+
+  <section v-else>
+    <div v-if="loading">Loading...</div>
+
+    <div v-else v-for="currency in info" class="currency">
+      {{ currency.description }}:
+      <span class="lighten">
+        <span v-html="currency.symbol"></span>{{ currency.rate_float | currencydecimal }}
+      </span>
+    </div>
+
+  </section>
+</div>
+```
+
+You can hit the rerun button on this pen to see the loading status briefly while we gather data from the API:
+
+<p data-height="300" data-theme-id="32763" data-slug-hash="6c01922c9af3883890fd7393e8147ec4" data-default-tab="result" data-user="Vue" data-embed-version="2" data-pen-title="Fourth Step Axios and Vue" class="codepen">See the Pen <a href="https://codepen.io/team/Vue/pen/6c01922c9af3883890fd7393e8147ec4/">Fourth Step Axios and Vue</a> by Vue (<a href="https://codepen.io/Vue">@Vue</a>) on <a href="https://codepen.io">CodePen</a>.</p>
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
+This can be even futher improved with the use of components for different sections and more distinct error reporting, depending on the API you're using and the complexity of your application.
+
+## Alternative Patterns
+
+### Fetch API
+
+The [Fetch API](https://developers.google.com/web/updates/2015/03/introduction-to-fetch) is a powerful native API for these types of requests. You may have heard that one of the benefits of the Fetch API is that you don't need to load an external resource in order to use it, which is true! Except... that it's not fully supported yet, so you will still need to use a polyfill. There are also some gotchas when working with this API, which is why many prefer to use axios for now. This may very well change in the future though.
+
+If you're interested in using the Fetch API, there are some [very good articles](https://scotch.io/@bedakb/lets-build-type-ahead-component-with-vuejs-2-and-fetch-api) explaining how to do so.
+
+## Wrapping Up
+
+There are many ways to work with Vue and axios beyond consuming and displaying an API. You can also communicate with Serverless Functions, post/edit/delete from an API where you have write access, and many other benefits. Due to the straightforward integration of these two libraries, it's become a very common choice for developers who need to integrate HTTP clients into their workflow.
```

Comment:
Sorry for late but does "post/edit/delete" here mean the [HTTP methods](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html)? If yes, the method "edit" should be "put" I guess.
Thanks.